pages/profile/read/views/index.jsx

```diff
@@ -1,276 +1,18 @@
-                                        <Fragment>{module.species}<br /></Fragment>
-                                      )
-                                    }
-                                    <Snippet date={formatDate(module.passDate)}>training.dateCompleted</Snippet>
-                                  </dd>
-                                </Fragment>
-                              )
-                            }
-                          </dl>
-                        </dd>
-                      </dl>
-                    )
-                  }
-                </ExpandingPanel>
-              )
-            }
-            {
-              (address || telephone || email) && (
-                <ExpandingPanel title={<Snippet>contactDetails.title</Snippet>}>
-                  <dl className="inline">
-                    {
-                      address && (
-                        <Fragment>
-                          <dt><Snippet>contactDetails.professionalAddress</Snippet></dt>
-                          <dd>{address}<br />{postcode}</dd>
-                        </Fragment>
-                      )
-                    }
-                    {
-                      telephone && (
-                        <Fragment>
-                          <dt><Snippet>contactDetails.telephone</Snippet></dt>
-                          <dd>{telephone}</dd>
-                        </Fragment>
-                      )
-                    }
-                    {
-                      email && (
-                        <Fragment>
-                          <dt><Snippet>contactDetails.email</Snippet></dt>
-                          <dd><a href={`mailto:${email}`}>{email}</a></dd>
-                        </Fragment>
-                      )
-                    }
-                  </dl>
-                </ExpandingPanel>
-              )
-            }
-            <ExpandingPanel title={<Snippet role={profileRole}>permissionLevel.title</Snippet>}>
-              <dl className="inline">
-                {
-                  !isUser && allowedActions.includes('profile.permissions') && (
-                    <Fragment>
-                      <dt>{profileRole}</dt>
-                      <dd><Link page="profile.permission" label={<Snippet>pages.profile.permission.change</Snippet>} /></dd>
-                    </Fragment>
-                  )
-                }
-              </dl>
-            </ExpandingPanel>
-          </Accordion>
-        </div>
-      </article>
-    </Fragment>
+    <Profile establishment={establishment} profile={model} allowedActions={allowedActions} title={establishment.name}/>
   );
 };
 
-const mapStateToProps = ({ static: { establishment, isUser, profileRole, allowedActions }, model }) => ({ establishment, model, isUser, profileRole, allowedActions });
+const mapStateToProps = ({ static: { establishment, profileRole, allowedActions }, model }) => ({ establishment, model, profileRole, allowedActions });
```

Comment:
you have left `profileRole` here and it isn't being used anymore, sorry @lvnhmd, please don't hate me :)